cmd/deploy/validate_test.go

```diff
@@ -65,7 +65,9 @@ func Test_validateAndSet(t *testing.T) {
 				return nil
 			}
 
-			err := validateAndSet(tt.variables, setEnvStorage)
+			opts := &Options{Variables: tt.variables}
+
+			err := validateAndSetVariables(opts, setEnvStorage)
```

Comment:
Can we test `cleanEscapedVariables`

Reply:
refactored. Done 4e2b7b8ee0f399d8715119761e4b511797e8d6b6